### PR TITLE
Diagnostic Report Enhancements

### PR DESCRIPTION
**Pull request by damonbayer**
Closes https://github.com/CDCgov/pyrenew-hew/issues/108

### CODE REVIEW

**Comment by damonbayer** [comment]
@dylanhmorris I am still working on the workflow parts of this to get things rendered in Azure, but you can review the current output in pyrenew-test-output/influenza_r_2024-11-19_f_2024-08-16_t_2024-11-13/diagnostic_report` and comment here if you have any feedback.

**Comment by dylanhmorris** [comment]
> @dylanhmorris I am still working on the workflow parts of this to get things rendered in Azure, but you can review the current output in pyrenew-test-output/influenza_r_2024-11-19_f_2024-08-16_t_2024-11-13/diagnostic_report and comment here if you have any feedback.

The example site looks really nice. Thanks, @damonbayer! LMK when the PR is ready for review.

**Comment by damonbayer** [comment]
Azure bugs are fixed. Code is ready for your review @dylanhmorris

File: pipelines/render_webpage.R [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+  file_delete(page_qmd)
+}
+
+p <- arg_parser("Render diagnostic dashboard for a single location forecast") |>
+  add_argument(
+    "model_run_dir",
+    help = "Directory containing the model data and output."
+  ) |>
+  add_argument("--template_qmd_path",
```
Comment (dylanhmorris) [inline pipelines/render_webpage.R:51]:
```suggestion
  add_argument("--template-qmd-path",
```

**Review by dylanhmorris — APPROVED** [review]
Approved. Tiny command line argument style point.

**Pull request merged**